lambdas/service/app.py

```diff
@@ -1284,28 +1263,27 @@ def repository_id_spec(index_name_singular: str):
     }
 
 
-def repository_head_spec(index_name):
-    search_spec_link = f'#operations-Index-get_index_{index_name}'
+def repository_head_spec(for_summary: bool = False):
+    search_spec_link = f'#operations-Index-get_index_{"summary" if for_summary else "_entity_type_"}'
```

Comment:
This generates two consecutive underscores if `for_summary` is False. Is that what you intend?

Reply:
Yes. Path variables such as `{entity_type}` are interpolated into the links by replacing the braces with underscores, resulting in double or potentially triple underscores. I verified all the internal links were functional.